components/treeview/index.css

```diff
@@ -36,8 +39,9 @@ governing permissions and limitations under the License.
   --spectrum-treeview-item-background-color-focus: var(--spectrum-gray-200);
 
   --spectrum-treeview-item-border-color-quiet-selected: transparent;
-  
+
   --spectrum-treeview-item-icon-gap: var(--spectrum-text-to-visual-75);
+  --spectrum-treeview-item-icon-offset-block: -1px;
```

Comment:
Just to double-check this custom property: it looks like `sizeM` is the "default", and medium is the only instance where we'd use `-1px`, right?

Reply:
Yes, but I've gone ahead and removed this custom property entirely with the latest refactor. Which simplifies the layout to remove the need for all of the negative block margins and offsets.